WikiDrips/WikiTableViewController.swift

```diff
@@ -148,6 +148,7 @@ class WikiTableViewController: UITableViewController {
         }
         
         let wikiDoc = wikiDocs[indexPath.section][indexPath.row]
+        wikiCell.wikiDoc = wikiDoc
```

Comment:
Hmm, I don't like the 'truth' being in a whole bunch of places, specially here, you're adding a model object to a view. MVC usually keeps these things separate. Besides, the cell doesn't need this property at all, its a cell, not a 🐴  for it to take data to other parts of your code :)

On line 127, you just grab the title from `wikiDocs` directly using `indexPathForCell` method on table view.